torch/csrc/utils/python_arg_parser.h

```diff
@@ -520,6 +521,11 @@ inline std::string PythonArgs::string(int i) {
   return THPUtils_unpackString(args[i]);
 }
 
+inline c10::optional<std::string> PythonArgs::stringOptional(int i) {
```

Comment:
`inline c10::optional<std::reference_wrapper<const std::string>> PythonArgs::stringOptional(int i) {`

Reply:
That would cause a dangling reference.